Scripts/script-ExtractEmail.yml

```diff
@@ -8,7 +8,9 @@ script: |
       text = JSON.stringify(args.text).replace(/\\n/g,' ');
   }
   var matches = {};
-  var reg = /[a-z0-9]+[_a-z0-9\.-]*[a-z0-9]+@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})/gmi;
+  var emailStringRegex = emailRegex.toString().substring(1,72);
```

Comment:
in substring use `length` of `emailRegex` instead of 72, if the regex will change no one will remember to fix it here